modules/sbo-advanced-binding-options.adoc

```diff
@@ -115,4 +121,66 @@ spec:
 <1> Name of the backing service resource.
 <2> Optional identifier.
 <3> Generated JSON name that is to be projected as the file content or environment value. The JSON name contains the attributes of the backing service custom resource.
-<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
\ No newline at end of file
+<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
+
+[id="binding-workloads-using-a-label-selector_{context}"]
+== Binding workloads using a label selector
+You can use a label selector to specify the workload that is being bound. If you declare a service binding using the label selectors to pick up workloads, the {servicebinding-title} periodically attempts to find and bind new workloads that match the given label selector.
```

Comment:
How about:
```suggestion
You can use a label selector to specify the workload to bind. If you declare a service binding using the label selectors to pick up workloads, the {servicebinding-title} periodically attempts to find and bind new workloads that match the given label selector.
```
This suggestion incorporates the following guidelines:
- Avoid passive voice.